nncf/quantization/algorithms/bias_correction/openvino_backend.py

```diff
@@ -126,6 +122,8 @@ def get_output_name(model: ov.Model, node_name: str) -> str:
 
     @staticmethod
     def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        if node.layer_attributes is None:
```

Comment:
For which nodes need check `node.layer_attributes`?

Reply:
See the code in a few lines below.
Also, since `is_quantized_weights` is called for each node in the algorithm, we need to check that `layer_attributes` exists.